lib/vsc/ldap/user.py

```diff
@@ -102,33 +104,37 @@ def modify_ldap(self, attributes):
         """
         self.ldap_query.user_modify(self.user_id, attributes)
 
-    def get_group(self, reload=False):
+    def get_group(self, do_reload=False):
```

Comment:
yeah, it's a bit of a pickle here...
I guess we could do
```
def get_group(self, reload=reload, do_reload=False):
    if reload != reload:
        logging.warning('you are redefining the built in reload, this argument is deprecated, use do_reload instead!')
```
but this would still allow functions calling us to redifine reload, so we should als del it instantly?